tests/TestYlabel.cpp

```diff
@@ -0,0 +1,89 @@
+#include "trase.hpp"
+#include <iostream>
+#include <fstream>
+
+using namespace trase;
+
+void test_1() {
```

Comment:
can you please integrate these two tests into the `TestAxis.cpp` file. And add some explicit checks, e.g. check that `m_max_ytick_len` varies between the two cases in the way you expect